source/common/runtime/runtime_features.cc

```diff
@@ -45,6 +45,41 @@ constexpr const char* disallowed_features[] = {
     // Acts as both a test entry for deprecated.proto and a marker for the Envoy
     // deprecation scripts.
     "envoy.deprecated_features.deprecated.proto:is_deprecated_fatal",
+    // 1.12.0
+    "envoy.deprecated_features.route.proto:regex_match",
+    "envoy.deprecated_features.route.proto:regex",
+    "envoy.deprecated_features.accesslog.proto:config",
+    "envoy.deprecated_features.thrift_proxy.proto:config",
+    "envoy.deprecated_features.cds.proto:tls_context",
+    "envoy.deprecated_features.route.proto:allow_origin",
+    "envoy.deprecated_features.string.proto:regex",
+    "envoy.deprecated_features.grpc_service.proto:config",
+    "envoy.deprecated_features.tcp_proxy.proto:deprecated",
```

Comment:
Is this right? I think this should be "deprecated_v1"

Reply:
Yeah, the script doesn't deal well with 

// [#next-free-field: 11]
message TcpProxy {
  // [#not-implemented-hide:] Deprecated.
  // TCP Proxy filter configuration using V1 format.
  message DeprecatedV1 {
    option deprecated = true;

I think this one is beyond python so should manually tweak.